client/src/accounts/authentication/login.hook.test.tsx

```diff
@@ -0,0 +1,68 @@
+import { describe, expect, it } from 'vitest';
+import { HTTP_BAD_REQUEST, HTTP_OK } from '~/api/api.constants';
+import { rest, server } from '~/mocks/server';
+import { act, renderHook, waitFor } from '~/test/utils';
+import { LoginForm } from './login-form.component';
+import { useLogin } from './login.hook';
+const ENDPOINT = '*/api/accounts/';
+
+interface Result {
+  isSuccess: boolean;
+  isError: boolean;
+  data: Record<string, unknown>;
+  mutate: (form: Record<string, unknown>) => void;
+  error: Record<string, unknown>;
+  current: Record<string, unknown>;
+}
+
+describe('useLoginHook', () => {
+  it('should throw an error for an unknown user', async () => {
+    const loginData = { email: 'bob@example.com', password: 'otherpassword', accessToken: 'foobar' };
+    server.use(rest.post(ENDPOINT, (req, res, ctx) => res(ctx.status(200), ctx.json({ message: 'User not found' }))));
+    const { result } = renderHook<Result, unknown>(() => useLogin());
+
+    act(() => {
+      result.current.mutate(loginData);
+    });
+    await waitFor(() => expect(result.current.isSuccess).toBe(false));
```

Comment:
Try be consistent in your code, there is no whitespace here, but there is in other tests, such quirks are easy to introduce, we all do it, but this is why you have to create a checklist for yourself to help you catch the silly mistakes you personally make.